pkg/database/sqlite/config.go

```diff
@@ -1,17 +1,44 @@
 package sqlite
 
+import (
+	"os"
+	"path/filepath"
+)
+
 // Config implements database.Configurator interface and
 // handles the default configuration parameters of the sqlite database.
 type Config struct {
 	Path string
+	Home string
 }
 
 func (c *Config) SetDefaults() {
-	if c.Path == "" {
-		c.Path = "storage.db"
+	if c.Path != "" {
+		return
 	}
+
+	c.Path = DefaultPath(c.Home)
 }
 
 func (c *Config) Validate() error {
 	return nil
 }
+
+// DefaultPath returns the default sqlite path for the given home directory.
+func DefaultPath(home string) string {
+	if home == "" {
+		home = os.Getenv("TERRALIST_HOME")
+	}
```

Comment:
This is not necessary. `viper` makes sure to automatically fallback to the environment variables if the configuration file / cli arguments does not provide this value.